pykern/pkio.py

```diff
@@ -128,6 +129,21 @@ def has_file_extension(filename, to_check):
     return e in to_check
 
 
+def is_pure_text(filename, test_size=512):
+    """Uses pykern.util.is_pure_text to guess whether file is pure text
```

Comment:
Explain this without using "pure_text". You can say using is_pure_text, but then you might say: 

Read test_size bytes of path to determine if it is text file

See `pykern.util.is_pure_text` for the details on the heuristic used to test the first text_size bytes.